hagiyama/Direction.cpp

```diff
@@ -0,0 +1,73 @@
+#include"Direction.h"
+#include"iostream"
+using namespace std;
+
+int main(void){
+  enum class Direction{N,NE,E,SE,S,SW,W,NW};
+  struct Coordinate{
+  private:
+    int x;
+    int y;
+  public:
+    int calcNextDirection(Coordinate &now,Coordinate &next){

```

Comment:
```suggestion
    int calcNextDirection(Coordinate &next){

```
にして，now.xとかnow.yはメンバ変数のxとyを使えばいいんじゃない？